handlers/contact-us-api/src/main/scala/com/gu/contact_us_api/contactUsApp.scala

```
@@ -0,0 +1,31 @@
+package com.gu.contact_us_api
+
+import com.gu.contact_us_api.models.SFErrorDetails
+import io.circe.parser._
+import io.circe.generic.auto._
+import io.circe.syntax._
+
+object contactUsApp {
+  def main(args: Array[String]): Unit = {
+    val json: String =
+      """
+      {
+        "topic": "billing",
+        "subtopic": "s2",
+        "subsubtopic": "ss4",
+        "name": "Manuel Joaquim",
+        "email": "manuel.joaquim@email.com",
+        "subject": "Extra charges",
+        "message": "EXTRA CHARGES OMGWTFBBQ!!1",
+        "attachment": {
+          "name": "printscreen.jpeg",
+          "contents": "junkfornow"
+        }
+      }
+    """
+
+    val contactUsReqHandler = new ContactUs(new SalesforceConnector())
+
+    println(contactUsReqHandler.processReq(json).asJson)
```

Make sure there are no silent errors. Sometimes this happens when errors are being modelled as pure values instead of exceptions.

---

I'm going to handle that when I introduce logging and error handling. If there is an error I still need to send a response to the client to report it so I can't just throw them. My plan is to handle the error just before I convert it into a Success/Failure response in `ContactUs.processReq`, which is why I'm returning `Either` up to that point.